mlflow/models/cli.py

```diff
@@ -156,19 +156,21 @@ def prepare_env(
 
 
 @commands.command("build-docker")
-@cli_args.MODEL_URI
+@cli_args.MODEL_URI_BUILD_DOCKER
 @click.option("--name", "-n", default="mlflow-pyfunc-servable", help="Name to use for built image")
 @cli_args.ENV_MANAGER
 @cli_args.MLFLOW_HOME
 @cli_args.INSTALL_MLFLOW
 @cli_args.ENABLE_MLSERVER
 def build_docker(model_uri, name, env_manager, mlflow_home, install_mlflow, enable_mlserver):
     """
+    If ``--model-uri`` is specified:
```

Comment:
Can we combine the "if model-uri is specified" and "if model uri is not specified" parts instead of separating them?

We can say something like:

Builds a Docker image whose default entrypoint serves an MLflow model at port 8080, using the `python_function` flavor. The container serves the model referenced by `model_uri`, if specified when `build_docker` is called. If `model_uri` is not specified when `build_docker` is called, an MLflow Model directory must be mounted as a volume into the ``/opt/ml/model`` directory in the container, as shown in the example below.

Reply:
Sure!